.gitignore

```diff
@@ -7,7 +7,7 @@ bin
 .vscode
 .mypy_cache
 
-*/.gradle
-*/.vscode
-*/.wpilib
-*/build
\ No newline at end of file
+**/.gradle
+**/.vscode
+**/.wpilib
+**/build
```

Comment:
I think if we leave off the `**/` it'll work as expected too.

Reply:
Hmm, without the second asterisk, Git resolves those folders as untracked. With the double asterisks, it ignores them.

This screenshot was with a single asterisk: http://prntscr.com/oaloid

Reply:
That's why I'm saying leave off the globs entirely.

Reply:
Ah yes, that does work. Fixed in 4752beb